htdocs/core/class/html.form.class.php

```diff
@@ -6511,19 +6511,19 @@ public function load_tva($htmlname = 'tauxtva', $selectedrate = '', $societe_ven
 		}
 		if (getDolGlobalString('SERVICE_ARE_ECOMMERCE_200238EC')) {    // If option to have vat for end customer for services is on
 			require_once DOL_DOCUMENT_ROOT . '/core/lib/company.lib.php';
-			if (!isInEEC($societe_vendeuse) && (!is_object($societe_acheteuse) || (isInEEC($societe_acheteuse) && !$societe_acheteuse->isACompany()))) {
+			if (isInEEC($societe_vendeuse) && isInEEC($societe_acheteuse) && !$societe_acheteuse->isACompany()) {
 				// We also add the buyer country code
 				if (is_numeric($type)) {
 					if ($type == 1) { // We know product is a service
-						$code_country .= ",'" . $societe_acheteuse->country_code . "'";
+						$code_country = "'" . $societe_acheteuse->country_code . "'";
```

Comment:
When SERVICE_ARE_ECOMMERCE_200238EC is set, we want to have ALSO in the combo list the vat rate of the purchaser country because there is a lot of exception and restricting the list to purchaser country only will block user if he need to fallback to its own vat rate. This is why we did a concat .=

Reply:
Ok, I think it's the right way. But where should I document this new value for SERVICE_ARE_ECOMMERCE_200238EC ?